elements/rh-tabs/rh-tab.ts

```diff
@@ -92,13 +89,13 @@ export class RhTab extends LitElement {
     const last = lastTab === this;
     return html`
       <div id="container" class="${classMap({ active, box, vertical, first, last, [on]: !!on })}">
-        <button part="button" ?disabled="${this.disabled}">
+        <div id="button" part="button" ?disabled="${this.disabled}">
```

Comment:
I wonder if we could combine this with the container element?

Reply:
when you get the opportunity PTAL again!